caikit/core/__init__.py

```diff
@@ -31,6 +31,7 @@
 
 # Local
 from .data_model import DataObjectBase, dataobject
+from .exceptions import DataValidationError, error_handler
```

Comment:
Moving `error_handler` out of `toolkit` will break a _looooooot_ of downstreams

Reply:
Interestingly, we have a unit test that asserts that `error_handler` is importable from `caikit.core` that seemed to suggest that was the way that users should access it: https://github.com/caikit/caikit/blob/main/tests/core/test_imports.py#L16-L21